test/python/notebooks/test_pbar_status.ipynb

```diff
@@ -21,7 +21,8 @@
    "source": [
     "from qiskit import LegacySimulators, QuantumRegister, ClassicalRegister, QuantumCircuit, execute, compile\n",
     "from qiskit.tools.parallel import parallel_map\n",
-    "from qiskit.tools import job_monitor, TextProgressBar\n",
+    "from qiskit.tools.monitor import job_monitor\n",
+    "from qiskit.tools.events import TextProgressBar\n",
```

Comment:
I find a bit weird having a ProgressBar under an `events` directory. I thought we were moving it to just `tools`.

Reply:
I was not changing location. I think it is good to have all the tools contained so that they are clear what they are. I am good with a follow on pr moving things around.  

I also don’t like events but we could not think of a better name.

Reply:
This was an init file that linked to this directory.